app/docker/networks/edit/NetworkOptionsTable.tsx

```diff
@@ -0,0 +1,33 @@
+import { Widget, WidgetBody, WidgetTitle } from '@/portainer/components/widget';
+import { DetailsTable } from '@/portainer/components/DetailsTable';
+
+import { NetworkOptions } from '../types';
+
+type Props = {
+  options: NetworkOptions;
+};
+
+export function NetworkOptionsTable({ options }: Props) {
+  if (Object.keys(options).length === 0) {
+    return null;
+  }
+
+  return (
+    <div className="row">
+      <div className="col-lg-12 col-md-12 col-xs-12">
+        <Widget>
+          <WidgetTitle title="Network options" icon="fa-cogs" />
+          <WidgetBody className="nopadding">
+            <DetailsTable dataCy="networkDetails-networkOptionsTable">
+              {Object.entries(options).map(([key, value]) => (
+                <DetailsTable.Row key={key} label={key}>
+                  {value}
+                </DetailsTable.Row>
+              ))}
+            </DetailsTable>
+          </WidgetBody>
+        </Widget>
+      </div>
+    </div>
```

Comment:
this still calls `keys` and `entries`. to reduce this calculation would be easier if we have
```
const entries = Object.entries(options)
if (entries.length) {
  return null
}

<Widget>
    {entries.map(([key, value]) => .... }
```